README.md

```diff
@@ -1732,7 +1732,9 @@ _**Unofficial** set of patterns for structuring projects._
 * [ego](https://github.com/benbjohnson/ego) - Lightweight templating language that lets you write templates in Go. Templates are translated into Go and compiled.
 * [extemplate](https://github.com/dannyvankooten/extemplate) - Tiny wrapper around html/template to allow for easy file-based template inheritance.
 * [fasttemplate](https://github.com/valyala/fasttemplate) - Simple and fast template engine. Substitutes template placeholders up to 10x faster than [text/template](http://golang.org/pkg/text/template/).
+* [fet](https://github.com/fefit/fet) - FET is a golang template engine that can tranlate smarty like template code into golang html/template.
```

Comment:
the process of adding a new package to the list is different from the process of updating (typo), I need you to remove this link and create a new pull request

Reply:
Why is it so much more complicated than a simple push?